module/documents/actor.mjs

```diff
@@ -76,27 +81,50 @@ export default class ActorEd extends Actor {
 
   /**
    * Legend point History earned prompt trigger
+   * @returns {Promise<void>}
    */
-  async legendPointHistoryEarned( ) {
+  async legendPointHistoryEarned() {
     // let history = await getLegendPointHistoryData( actor );
-    const lpUpdateData = await LegendPointHistoryEarnedPrompt.waitPrompt( new LpTrackingData( this.system.lp.toObject() ), {actor: this} );
-    return this.update( {system: { lp: lpUpdateData }} )
+    const lpUpdateData = await LegendPointHistoryEarnedPrompt.waitPrompt( new LpTrackingData( this.system.lp.toObject() ), { actor: this } );
+    return this.update( { system: { lp: lpUpdateData } } )
```

Comment:
The doc says it returns `Promise<void>` which is not true I think. It's the updated document iirc.

Reply:
ok, unsure what this should be

Reply:
Promise<void> returns null or undefined...